schemas/2.2/schema.json

```
@@ -21,6 +21,22 @@
         "2.2"
       ]
     },
+    "instance":{
+      "description": "Metadata about the instance",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "description": "The utf8 encoded name",
+          "type": "string"
+        },
+        "description": {
+          "description": "This should describe the instance and might also be markdown formated",
```

If we really want to allow formatting here then I'd strongly vote for requiring well defined and standardized one, such as CommonMark. However I'm not super comfortable with allowing this at all. Unless very carefully configured most Markdown/CommonMark or similar converters allow arbitrary HTML and thus we're asking especially aggregators to implement XSS footguns with that. While probably not a high value target I don't think it's really worth opening that door.

So I'd vote for no formatting being allowed for simplicities sake, alternatively let's pick a well-restricted simple formatting standard.

```suggestion
          "description": "If supported by the software the administrator configured long form description of this instance",
```